src/main/java/Duke.java

```diff
@@ -1,10 +1,142 @@
+        String[] paramAndDate = splitInputLine(commandTypeAndParams[1], "/");
+        String commandWord = commandTypeAndParams[0];
+        String commandArgs = paramAndDate[0];
+        String commandDate = paramAndDate[1];
+        
+        switch(commandWord.toLowerCase()) {
+        case COMMAND_TODO_WORD:
+            tasks[Task.getTaskCount()] = new Todo(commandArgs);
+            displayAddMessage();
+            break;
+        case COMMAND_DEADLINE_WORD:
+            tasks[Task.getTaskCount()] = new Deadline(commandArgs, commandDate);
+            displayAddMessage();
+            break;
+        case COMMAND_EVENT_WORD:
+            tasks[Task.getTaskCount()] = new Event(commandArgs, commandDate);
+            displayAddMessage();
+            break;
+        case COMMAND_LIST_WORD:
+            displayTaskList();
+            break;
+        case COMMAND_DONE_WORD:
+            displayDoneMessage(commandArgs);
+            break;
+        case COMMAND_BYE_WORD:
+            displayByeMessageAndExit();
+            break;
+        case COMMAND_BLAH_WORD:
+            displayBlahMessage();
+            break;
+        default:
+            System.out.println(DIVIDER + LS + COMMAND_INVALID_MESSAGE + LS + DIVIDER);
+        }
+    }
+    
+    private static String getUserInput() {
+        System.out.print("Enter a command: ");
+        String inputLine = in.nextLine();
+        echoUserInput(inputLine.trim());
+        return inputLine.trim();
+    }
+    private static void echoUserInput(String userInput) {
+        System.out.println("Command entered: " + userInput);
+    }
+    //Solution below adapted from https://github.com/nus-cs2113-AY1920S2/contacts/blob/master/src/main/java/Contacts1.java
+    public static String[] splitInputLine(String rawUserInput, String regex) {
+        String[] split = rawUserInput.trim().split(regex, 2);
+        return split.length == 2? split : new String[] {split[0], ""}; // else no parameters
+    }
+    
+    private static void displayWelcomeMessage() {
+        System.out.println(DIVIDER + LS + WELCOME_MESSAGE + LS + DIVIDER);
+    }
+    
+    private static void displayAddMessage() {
+        System.out.println(DIVIDER);
+        System.out.println(String.format(COMMAND_ADD_MESSAGE, tasks[Task.getTaskCount() -1], Task.getTaskCount()));
+        System.out.println(DIVIDER);
+    }
+    
+    private static void displayDoneMessage(String index) {
+        int doneTaskIndex =  Integer.parseInt(index) - 1;
+        if(Task.getTaskCount() <= doneTaskIndex){
+            System.out.println(DIVIDER + LS + COMMAND_INVALID_INDEX + LS + DIVIDER);
+        } else {
+            tasks[doneTaskIndex].markAsDone();
+            System.out.println(DIVIDER + LS + COMMAND_DONE_MESSAGE);
+            System.out.println(TAB + tasks[doneTaskIndex]);
+            System.out.println(DIVIDER);
+        }
+    }
+    
+    private static void displayBlahMessage() {
```

Comment:
Is this method a debug method? By removing this method, does it improve readability?